src/tests/test_enstore_plotter_module.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+import enstore_plotter_module
+import os
+import errno
+import time
+import tempfile
+
+class TestEnstorePlotterModule(unittest.TestCase):
+
+    def setUp(self):
+        self.epm = enstore_plotter_module.EnstorePlotterModule("test")
+
+    def test___init__(self):
+        self.assertTrue(self.epm.name == "test")
+        self.assertTrue(isinstance(self.epm, enstore_plotter_module.EnstorePlotterModule))
+
+    def test_set_and_isActive(self):
+        self.assertEqual(self.epm.isActive(), True) 
+        self.epm.setActive(False)
+        self.assertEqual(self.epm.isActive(), False)
+
+    def test_book(self):
```

Comment:
These four look a bit strange, I guess they're just testing that the functions they "test" don't raise errors, but that seems like the opposite behavior from what I would expect for functions that are intended to be overwritten by subclasses -- I think the "correct" think to do would be to throw errors in these functions and test for that, but it is probably not worth adding failure cases to the codebase... 

I think what might make it a bit more clear would be to change all these functions (book, fill, plot, install) to simply `pass` and comment in the tests as well that they're all noops intended to be overwritten. WDYT?